TFNetworkRecLayer.py

```diff
@@ -720,6 +722,15 @@ def is_prev_step_layer(cls, layer):
       return layer.is_prev_time_frame
     return False
 
+  def get_sub_layer(self, layer_name: str):
+    """
+    :param layer_name:  The sub_layer addressed by '/' separated path.
+    :return: The sub_layer addressed in layer_name or None if no sub_layer exists
+    """
+    if '/' in layer_name and layer_name.split('/')[0] == 'unit':
```

Comment:
Why this logic? Why only if it is `unit`? I would not do that, but rather for any `layer_name`.
But instead, check `isinstance(self.cell, _SubnetworkRecCell)`, because only in that case you want that.